core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -207,6 +207,15 @@ private TableProperties() {}
   public static final String SPLIT_SIZE = "read.split.target-size";
   public static final long SPLIT_SIZE_DEFAULT = 128 * 1024 * 1024; // 128 MB
 
+  public static final String ADAPTIVE_SPLIT_SIZE_ENABLED = "read.split.adaptive-size.enabled";
+  public static final boolean ADAPTIVE_SPLIT_SIZE_ENABLED_DEFAULT = false;
+
+  public static final String ADAPTIVE_SPLIT_MIN_SIZE_BYTES = "read.split.min-adaptive-size-bytes";
```

Comment:
Another way to name these properties like this:

```
read.split.adaptive-size.enabled
read.split.adaptive-size.min-value-bytes
read.split.adaptive-size.max-value-bytes
```

Reply:
I think I prefer the alternative names.

Reply:
Renamed.